DSharpPlus/Exceptions/RequestSizeException.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Newtonsoft.Json.Linq;
+using DSharpPlus.Net;
+
+namespace DSharpPlus.Exceptions
+{
+    /// <summary>
+    /// Represents an exception thrown when the request sent to Discord is too large.
+    /// </summary>
+    class RequestSizeException : Exception
```

Comment:
I believe this class should be made public.

Reply:
Oh whoops, I thought it already was.